tools/fetch_fasta_from_ncbi/retrieve_fasta_from_NCBI.xml

```diff
@@ -37,6 +55,15 @@
         <output name="outfilename" ftype="fasta" file="output.fa" />
         <!--  <output name="logfile" ftype="txt" file="log.txt" />  log.txt changes with timestamp. removed to pass the  test -->
     </test>
+    <test>
+        <param name="queryString" value="9629650[gi]" />
```

Comment:
Maybe you could use another ID for another test. It's the occasion to increase the scope of the tests !